api/BluetoothRemoteGATTCharacteristic.json

```diff
@@ -1116,7 +1116,101 @@
           },
           "status": {
             "experimental": true,
-            "standard_track": true,
+            "standard_track": false,
+            "deprecated": false
+          }
+        }
+      },
+      "writeValueWithoutResponse": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "85"
+            },
+            "chrome_android": {
+              "version_added": "85"
+            },
+            "edge": {
+              "version_added": "85"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
```

Comment:
Running https://mdn-bcd-collector.appspot.com/tests/api/BluetoothRemoteGATTCharacteristic in Opera 73 on macOS Catalina shows all features supported.